toolchest_client/tools/alphafold.py

```diff
@@ -0,0 +1,39 @@
+"""
+toolchest_client.tools.bowtie2
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This is the bowtie2 implementation of the Tool class.
+"""
+from toolchest_client.files import OutputType
+from datetime import date
+from . import Tool
+
+
+class AlphaFold(Tool):
+    """
+    The bowtie2 implementation of the Tool class.
+    """
+    def __init__(self, inputs, output_path, model_preset=None, max_template_date=None, use_reduced_dbs=False,
+                 is_prokaryote_list=None):
+        tool_args = (
+            (f"--model_preset={model_preset} " if model_preset is not None else "") +
+            (f"--max_template_date={max_template_date} " if max_template_date is not None
+             else f"--max_template_date={date.today().strftime('%Y-%m-%d')} ")  # +
+            # (f"--is_prokaryote_list={is_prokaryote_list.join(',')} " if is_prokaryote_list is not None else "") +
+            # ("--db_preset=reduced_dbs " if use_reduced_dbs else "")
+        )
```

Comment:
It looks like most tool_arg processing is happening in `api.py` for other tools.

Was this an opinionated choice? I don't have a preference.

Reply:
No I just started implementing these before adding AlphaFold to `api.py` I'll move it to `api.py` to match convention.